autode/conformers/conformer.py

```diff
@@ -2,16 +2,38 @@
 from autode.config import Config
 from autode.exceptions import AtomsNotFound
 from autode.log import logger
-from autode.species import Species
+from autode.species.species import Species
+
+
+def get_conformer(species, name):
+    """
+    Generate a conformer of a species given a new name
+
+    Args:
+        species (autode.complex.Complex):
+        name (str):
+
+    Returns:
+        (autode.conformers.Conformer)
+    """
+    conformer = Conformer(name=name, atoms=species.atoms,
+                          charge=species.charge, mult=species.mult)
+
+    # Conformer should be in the same solvent
+    conformer.solvent = species.solvent
+
+    return conformer
 
 
 class Conformer(Species):
 
-    def optimise(self, method):
+    def optimise(self, method, reset_graph=False):
```

Comment:
reset_graph doesn't do anything?

Reply:
For consistency with the species.optimise() it's there but I see what you mean